docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  backend:
+    build: ./backend
+    depends_on:
+      - db
+    ports:
+      - ${PORTS}
+    env_file:
+      - .env
+    container_name: backend
+    command: bash -c "export FLASK_ENV=development
+      && export FLASK_APP=app.py
+      && flask run --host 0.0.0.0 --port 8000"
```

Comment:
I think that port 8000 is set by default.

Reply:
flask sets port 5000 by default